.github/workflows/build-plugin-archive.yml

```
@@ -147,6 +149,11 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: ${{ needs.checkout-dependencies.outputs.artifact }}
+      - name: Setup PHP with tools
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ inputs.PHP_VERSION_BUILD }}
+          tools: humbug/php-scoper:dev-main
```

Is there a particular reason for using `dev-main`? Unreleased, buggy changes on that branch could lead to hard-to-detect bugs in the generated code.
```suggestion
          tools: humbug/php-scoper

```
Also added a new line above the next step.